client/components/searchBar.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+class SearchFood extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: ''
+    };
+    this.handleInput = this.handleInput.bind(this);
+    this.handleSearch = this.handleSearch.bind(this);
+  }
+
+  handleInput(event) {
+    const { value } = event.target;
+    this.setState({
+      inputValue: value
+    });
+  }
+
+  handleSearch(event) {
+    event.preventDefault();
+    fetch('https://trackapi.nutritionix.com/v2/search/instant?query=' + this.state.inputValue, {
+      method: 'GET',
+      headers: {
+        'x-app-id': process.env.REACT_APP_API_ID,
+        'x-app-key': process.env.REACT_APP_API_KEY
+      }
+    })
+      .then(response => response.json())
+      .then(input => this.state.inputValue)
+      .catch(error => console.error(error));
+  }
+
+  render() {
+
+    if (!this.state.inputValue) {
+      return (
+        <form
+          className="search"
+          onSubmit={this.handleSearch}>
+          <div className="searchbar">
+            <input
+              type="search"
+              placeholder="food search"
+              onChange={this.handleInput} />
+            <button className="search-button">
+              <i className="fas fa-search"></i>
+            </button>
+          </div>
+        </form>
+      );
+    } else {
+      return (
+        <>
+          <form
+            className="search"
+            onSubmit={this.handleSearch}>
+            <div className="searchbar">
+              <input
+                type="search"
+                placeholder="food search"
+                onChange={this.handleInput} />
+              <button className="search-button">
+                <i className="fas fa-search"></i>
+              </button>
+            </div>
+          </form>
+
+        </>
```

Comment:
I do not see the difference between these, is it possible to remove the `if`? Also, be sure to pass a `value` prop to your input, so that the `<input>` is "controlled".